Core/include/Acts/EventData/detail/MultiTrajectoryTestsCommon.hpp

```diff
@@ -471,8 +471,8 @@ class MultiTrajectoryTestsCommon {
         pc.sourceLink.sourceId);
     // Explicitly unset to avoid error below
     ts.unset(TrackStatePropMask::Calibrated);
-    testSourceLinkCalibratorReturn<trajectory_t>(
-        gctx, cctx, SourceLink{ttsb.sourceLink}, ts);
+    testSourceLinkCalibrator<trajectory_t>(gctx, cctx,
+                                           SourceLink{ttsb.sourceLink}, ts);
```

Comment:
Fair enough, but unrelated?

Reply:
Right this just popped up while I was fixing downstream issues. Should this be split out?

Reply:
Your call.